src/renderer/components/SyncView/SyncButton.js

```diff
@@ -16,7 +16,9 @@ const m = defineMessages({
   // Button when sync is complete
   complete: 'Complete',
   // Button to retry sync after error
-  retry: 'Retry'
+  retry: 'Retry',
+  // Disconnected
+  disconnected: 'Disconnected'
```

Comment:
Is this intended to be shown as a button label?

Reply:
Sme open questions here regarding how to display disconnected state. Right now it's greyed out, but I think it would be helpful to say 'disconnected' there.. will incorporate that and show a screen shot tomorrow

Reply:
![Screenshot from 2020-07-01 11-42-03](https://user-images.githubusercontent.com/633012/86301855-d250a380-bbbb-11ea-9659-5ece82adc175.png)

Reply:
![Screenshot from 2020-07-01 11-42-03](https://user-images.githubusercontent.com/633012/86306234-940db100-bbc8-11ea-929d-d095e7dee2e9.png)

Also now '100%...' when not finished yet will show 'Finishing...' instead